css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin-left: 15%;
+  margin-right: 15%;
```

Comment:
Rather than doing this, you might want to do `margin: auto`, which will center the content.